README.md

```diff
@@ -1,76 +1,54 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
 
-> One paragraph statement about the project.
+
+> Using linters
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
-## Built With
+#Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- VS Code
 
-## Live Demo
+Live Demo
 
 [Live Demo Link](https://livedemo.com)
 
 
```

Comment:
Please ensure that you customize the README.md to your project. Please remove the live demo or add the link to your project.